contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -29,6 +29,17 @@ using google::devtools::cloudtrace::v1::Traces;
 namespace google {
 namespace api_manager {
 
+RequestHandler::RequestHandler(ApiManagerImpl* api_manager,
+                               std::shared_ptr<CheckWorkflow> check_workflow,
+                               std::unique_ptr<Request> request_data)
+    : api_manager_(api_manager),
+      check_workflow_(check_workflow),
+      request_data_(std::move(request_data)) {
+  if (api_manager->IsConfigLoadingSucceeded()) {
```

Comment:
Hi @mangchiandjjoe 

With this case, I like to go back to your old way:   expose load_config_status in api_manager interface.    Let Nginx check it,  if still in progress,   go back to event loop to wait for it.  We will do this for both Check/Report and GrpcTranscoding.

I will like to keep the current pending_requests_callback code.   But in principal,  dead code should be removed.  If it is not used, it should be removed.